boards/arm/arduino_due/arduino_due_defconfig

```diff
@@ -11,3 +11,4 @@ CONFIG_SERIAL=y
 CONFIG_UART_SAM=y
 CONFIG_UART_SAM_PORT_0=y
 CONFIG_SOC_ATMEL_SAM3X_EXT_MAINCK=y
+CONFIG_WDT_DISABLE_AT_BOOT=y
```

Comment:
Should it be made conditional to WATCHDOG activation ?

Reply:
On Atmel SAM family the watchdog driver is enabled via a Kconfig default at the SoC level.